lib/backup_restore/restorer.rb

```diff
@@ -339,7 +339,6 @@ def switch_schema!
     def migrate_database
       log "Migrating the database..."
       Discourse::Application.load_tasks
-      ENV["VERSION"] = @current_version.to_s
```

Comment:
Not sure why we needed this since `current_version` is always point to the latest migration.